source/smart-contracts/references/host-fns.rst

```diff
@@ -56,89 +62,82 @@ Function parameter
 .. _host-functions-state:
 
 Smart contract instance state
-=================================================
+=============================
 
 .. function:: state_size() -> i32
 
    Get the byte size of the contract state.
 
-   :return: byte size of the contract state
+   :return: Byte size of the contract state.
    :rtype: i32
 
-.. function:: load_state(location, length, offset) -> i32
+.. function:: load_state(start, length, offset) -> i32
 
    Read a section of the state to the given location. Return the number of
    bytes written. The location is assumed to contain enough memory to write the
    requested length into. If not, the function will trap and abort execution of
    the contract.
 
-   :param i32 location: Pointer to write location.
-   :param i32 length: Number of bytes to read
-   :param i32 offset: Starting offset in the bytes
-   :return: The number of read bytes
+   :param i32 start: Pointer to write location in Wasm linear memory.
```

Comment:
I'm not a fan of these changes. These parameters are always the location where to write, which is why they were called that before. I'm not sure why there was a need to change that.